src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/AssetSchedules/IAssetScheduleRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Repositories;
+
+namespace EasyAbp.BookingService.AssetSchedules;
+
+public interface IAssetScheduleRepository : IRepository<AssetSchedule, Guid>
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="assetId"></param>
+    /// <param name="startingDateTime">Included</param>
+    /// <param name="endingDateTime">Excluded</param>
+    /// <param name="policy"></param>
+    /// <param name="includeDetails"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task<List<AssetSchedule>> GetAssetSchedulesAsync(Guid assetId, DateTime startingDateTime, DateTime endingDateTime,
```

Comment:
Maybe I misunderstood this method, but the implementation code is so complex. Should it rename to `GetAssetScheduleListInScopeAsync`? That may make people make sense without reading the code.